apps/next-app/src/pages/index.tsx

```diff
@@ -0,0 +1,175 @@
+import Image from 'next/image';
+import { Inter } from 'next/font/google';
+
+import { Button } from '@apps-monorepo/design-system';
+
+const inter = Inter({ subsets: ['latin'] });
+
+export default function Home() {
+  return (
+    <main
+      className={`flex min-h-screen flex-col items-center justify-between p-24 
+      ${inter.className}`}
+    >
+      <div
+        className="z-10 w-full max-w-5xl items-center justify-between
+        font-mono text-sm lg:flex"
+      >
+        <p
+          className="backdrop-blur- 2xl fixed left-0 top-0 flex w-full
+          justify-center border-b border-gray-300 bg-gradient-to-b from-zinc-200 pb-6
+          pt-8 dark:border-neutral-800 dark:bg-zinc-800/30 dark:from-inherit
+          lg:static lg:w-auto  lg:rounded-xl lg:border lg:bg-gray-200 lg:p-4
+          lg:dark:bg-zinc-800/30"
+        >
+          Get started by editing&nbsp;
+          <code className="font-mono font-bold">src/pages/index.tsx</code>
+        </p>
+        <div
+          className="fixed bottom-0 left-0 flex h-48 w-full items-end
+        justify-center bg-gradient-to-t from-white via-white dark:from-black
+        dark:via-black lg:static lg:h-auto lg:w-auto lg:bg-none"
+        >
+          <a
+            className="pointer-events-none flex place-items-center gap-2 p-8
+            lg:pointer-events-auto lg:p-0"
+            href="https://vercel.com?utm_source=create-next-app&utm_medium=
+            default-template-tw&utm_campaign=create-next-app"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            By
+            <Image
+              src="/vercel.svg"
+              alt="Vercel Logo"
+              className="dark:invert"
+              width={100}
+              height={24}
+              priority
+            />
+          </a>
+        </div>
+      </div>
+
+      <div
+        className="before:bg-gradient-radial after:bg-gradient-conic relative flex
+        place-items-center before:absolute before:h-[300px]
+        before:w-[480px] before:-translate-x-1/2 before:rounded-full
+        before:from-white before:to-transparent before:blur-2xl
+        before:content-[''] after:absolute after:-z-20 after:h-[180px]
+        after:w-[240px] after:translate-x-1/3 after:from-sky-200
+        after:via-blue-200 after:blur-2xl after:content-['']
+        before:dark:bg-gradient-to-br before:dark:from-transparent
+        before:dark:to-blue-700/10 after:dark:from-sky-900
+        after:dark:via-[#0141ff]/40 before:lg:h-[360px]"
+      >
+        <Image
+          className="relative dark:drop-shadow-[0_0_0.3rem_#ffffff70]
+          dark:invert"
+          src="/next.svg"
+          alt="Next.js Logo"
+          width={180}
+          height={37}
+          priority
+        />
+      </div>
+
+      <div className="mb-32 grid text-center lg:mb-0 lg:grid-cols-4 lg:text-left">
+        <a
+          href="https://nextjs.org/docs?utm_source=create-next-app&utm_medium=
```

Comment:
Sort the props. Always. Please.
Check the apps-suites config for this setup. You can look at `dealer-mobile/dev` for the latest config. This will have the linter call you our or do it for you on save.